Lib/test/test_asdl_parser.py

```diff
@@ -62,14 +62,14 @@ def test_product(self):
 
     def test_attributes(self):
         stmt = self.types['stmt']
-        self.assertEqual(len(stmt.attributes), 2)
+        self.assertEqual(len(stmt.attributes), 4)
         self.assertEqual(str(stmt.attributes[0]), 'Field(int, lineno)')
         self.assertEqual(str(stmt.attributes[1]), 'Field(int, col_offset)')
```

Comment:
Add tests for stmt.attributes[2] and stmt.attributes[3]?

Reply:
Added.